shared/constants/chat.js

```diff
@@ -177,6 +195,31 @@ export type UpdateLatestMessage = NoErrorTypedAction<'chat:updateLatestMessage',
 export type UpdateMetadata = NoErrorTypedAction<'chat:updateMetadata', {users: Array<string>}>
 export type UpdatedMetadata = NoErrorTypedAction<'chat:updatedMetadata', {[key: string]: MetaData}>
 
+export type EditMessage = NoErrorTypedAction<'chat:editMessage', {message: Message}>
+export type DeleteMessage = NoErrorTypedAction<'chat:deleteMessage', {message: Message}>
+export type ClickedAttach = NoErrorTypedAction<'chat:clickedAttach', {conversationIDKey: ConversationIDKey, filename: string, title: string}>
```

Comment:
`ClickedAttachment`?

Also I generally avoid using "click" since you don't have that on mobile.... "SelectedAttachment" might be better.